saleor/graphql/shop/mutations.py

```diff
@@ -83,6 +89,27 @@ def mutate(cls, root, info, input):
         return ShopDomainUpdate(shop=Shop(), errors=errors)
 
 
+class ShopFetchTaxRates(BaseMutation):
+    shop = graphene.Field(Shop, description='Updated Shop')
+
+    class Meta:
+        description = 'Fetch tax rates'
+
+    @classmethod
+    @permission_required('site.manage_settings')
+    def mutate(cls, root, info):
+        errors = []
+        try:
```

Comment:
Could we just check if the service is configured?

https://github.com/mirumee/django-prices-openexchangerates/blob/3f633ad20f62dce03c526e9af93335f2b6b1a950/django_prices_openexchangerates/tasks.py#L15-L18

The stack trace logger below will point to the above piece of code. `logger.exception` will make it look like the server crashed (it will appear as an error in Sentry, Datadog etc.).